core/web-assets/src/main/assets/js/apps/onms-resources/index.js

```diff
@@ -51,6 +52,7 @@ angular.module('onms-resources', [
   };
 
   $http.get('rest/resources?depth=0').success(function(data) {
+    $scope.loaded = true;
```

Comment:
It seems like you've missed the error handler in this case (to set `$scope.loaded=false`).

Reply:
Thanks @agalue for catching, added it